include/oneapi/tbb/flow_graph.h

```diff
@@ -3318,6 +3318,11 @@ class overwrite_node : public graph_node, public receiver<T>, public sender<T> {
         spin_mutex::scoped_lock l( my_mutex );
         if (my_buffer_is_valid && is_graph_active( my_graph )) {
             // We have a valid value that must be forwarded immediately.
+            if (dynamic_cast<continue_receiver*>(&s)) {
```

Comment:
I think we can use an existing method for this:

```suggestion
            if (s.is_continue_receiver()) {
```